src/components/settings/ContractVersion/index.tsx

```diff
@@ -42,6 +44,24 @@ export const ContractVersion = ({ isGranted }: { isGranted: boolean }) => {
           Unsupported contract
         </Typography>
       )}
+      <div style={{ marginTop: 15 }}>
+        {needsUpdate ? (
+          <Tooltip
+            title="Update now to take advantage of new features and the highest security standards available.
+You will need to confirm this update just like any other transaction."
+            placement="right-start"
+          >
+            <Typography variant="body2" style={{ display: 'flex', alignItems: 'center' }}>
+              <SvgIcon component={InfoIcon} inheritViewBox fontSize="small" color="border" style={{ marginRight: 4 }} />
+              Why should I upgrade?
+            </Typography>
+          </Tooltip>
+        ) : (
+          <Typography style={{ display: 'flex', alignItems: 'center' }}>
+            <CheckBoxIcon style={{ color: 'secondary', marginRight: 4 }} /> Latest version
```

Comment:
I suggest we use our primary colour for the checkmark. For which, it needs be applied directly to the icon:

```suggestion
            <CheckBoxIcon color="primary" sx={{ mr: 0.5 }} /> Latest version
```